pages/Auto-Anvil

```diff
@@ -0,0 +1,13 @@
+# Auto-Anvil
+
+# Obtaining
+
+The auto anvil is one of the more complicated crafting recipes. You can find it under the category "Energy & Electricity" on the second page.
+
+# Tiers
+
+There are 2 tiers of this machine. Tier 1 has a Repair Factor of 10%, Tier2 20%.
+
+# Usage
+
+Place the item you want to repair together with "Duct Tape" in the machine and wait. (You need energy for it to work)
```

Comment:
The other method of usage would be to combine enchantment books with items. Also link duct tape item if it does exist.